src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const money = 40;
+  const twenty = 20;
+  const fifty = 50;
+  const result = days * money;
+
+  if (days < 3) {
+    return result;
+  }
+
+  if (days < 7) {
+    return result - twenty;
+  }
+
+  return result - fifty;
```

Comment:
it'd be better to exchange like this
```suggestion
  if (days >= 7) {
    return rentalCost - proDiscount;
  }

  if (days >= 3) {
    return rentalCost - baseDiscount;
  }

  return rentalCost;
```